vue/components/ui/textarea-platforme/textarea-patforme.stories.js

```diff
@@ -0,0 +1,62 @@
+import { storiesOf } from "@storybook/vue";
+import { withKnobs, text, boolean, number } from "@storybook/addon-knobs";
+
+storiesOf("Input", module)
+    .addDecorator(withKnobs)
+    .add("Text Area", () => ({
+        props: {
+            id: {
+                default: text("Id", "textarea")
+            },
+            value: {
+                default: text("Value", "This is a text")
+            },
+            placeholder: {
+                default: text("Placeholder", "This is a placeholder")
+            },
+            descriptionLabel: {
+                default: text("Description text", "Description")
+            },
+            helpLabel: {
+                default: text("Help text", "Help or instruction text goes here")
+            },
+            disabled: {
+                default: boolean("Disabled", false)
+            },
+            initialWidth: {
+                default: number("Initial Width", 250)
+            },
+            initialHeight: {
+                default: number("Initial Height", 50)
+            },
+            resize: {
+                default: boolean("Resize", true)
+            }
+        },
+        data: function() {
+            return {
+                textAreaText: this.value
+            };
+        },
+        methods: {
+            textChanged(value) {
+                this.textAreaText = value;
+            }
+        },
+        template: `
+        <div>
+            <textarea-platforme
+                v-bind:id="id"
+                v-bind:value="value"
+                v-on:update:value="textChanged($event)"

```

Comment:
Use the `value` you're passing in the event emission rather than the event itself: `value => onValue(value)`